draft/spec/index.html

```diff
@@ -340,6 +340,47 @@ <h2>Object Structure</h2>
         ├── inventory.json
         └── inventory.json.sha512
         </pre>
+
+        <section id="object-interoperability">
+            <h2>Interoperability</h2>
+            <p>
+                Within an OCFL Object and its associated <a>Inventory</a>, the following restrictions are applied
+                to filesystem features. These restrictions are provided in an attempt to normalize file handling
+                behaviors and addressability of files within the Inventory.
+            </p>
+            <p>
+                If use of these features are required to preserve an exact copy of the content (for example,
+                preserving a hard-drive image), implementers SHOULD consider using a packaging format,
+                such as ZIP or TAR files or ISO disk images, to preserve the content. See the
+                [[OCFL-Implementation-Notes]] for further details.
+            </p>
+            <ul>
+                <li>
+                    <strong>Symbolic links</strong> (aka 'shortcuts' or 'aliases') MUST NOT be used within an
+                    OCFL Object. These are not portable and may lead to undefined behaviours in client applications
+                </li>
+                <li>References to <strong>file paths</strong> within the <a>inventory</a> MUST use the "forward-slash"
+                    ('/') as a path separator. Some filesystem use other characters such as the "back-slash" ('\')
+                    or "colon" (':') as a path delimiter.
+                </li>
+                <li>
+                    <strong>File permissions</strong> MAY be applied to files in an OCFL Object; however, implementers
+                    should be aware that these are not portable across filesystems, and so should not be relied on
+                    to protect sensitive content. This is also the case for extended filesystem flags, such as
+                    Access Control Lists or Hidden files.
+                </li>
+                <li>
+                    The <strong>character encoding</strong> of the filesystem and Inventory SHOULD be
```

Comment:
I don't know what it means to talk about character encodings of a **filesytem** -- I think our expectation here is bytestream fidelity, without that all is lost! I would thus rephrase this paragraph to talk only about inventory files